src/stateMachines/maintenance/syncDeviceInfo.ts

```diff
@@ -93,7 +93,8 @@ export class SyncDeviceInfo {
       fwSku: context.deviceInfo.sku,
       currentMode: context.deviceInfo.currentMode?.toString(),
       features: context.deviceInfo.features,
-      ipAddress: context.deviceInfo.ipConfiguration.ipAddress
+      ipAddress: context.deviceInfo.ipConfiguration.ipAddress,
```

Comment:
Don't we need to check for the existence of an IP address as we do in activation?